eng/pipelines/templates/Validate_Build_Output.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
```

Comment:
Should we do this in a separate PR to allow cherry-picking this change to servicing branches?